src/main/java/org/commonjava/cdi/util/weft/WeftPoolBoy.java

```diff
@@ -113,7 +113,15 @@ public void shutdown()
         }
     }
 
+    /**
+     * TODO: This may cause counter-intuitive sharing of thread pools for un-annotated injections...
+     */
```

Comment:
So does this mean this method is not recommended to use in future?

Reply:
We still use it. When we use @Inject, the ec always has name. I add the new api because 1. 'name' is important, and 2. ExecutorConfig is an annotation and difficult to use as normal interface to pass in the name.

Reply:
btw, the ec.name will override the dummy "weft-unannotated".

Reply:
I push a new commit to make it less confusing.